src/main/resources/templates/signup2.html

```diff
@@ -47,10 +52,10 @@ <h3 th:text="#{signup.createAcc.header}">Create Your Account</h3>
 											pwdwidget.MakePWDWidget();
 										</script>
 									</div>
-									<p>&nbsp;</p>
-	                                <div class="form-group">
-	                                    <input type="password" class="form-control" placeholder="Confirm Password" th:field="*{confirmPassword}"/>
-	                                </div>
+									<!--<p>&nbsp;</p>-->
+	                                <!--<div class="form-group">-->
+	                                    <!--<input type="password" class="form-control" placeholder="Confirm Password" th:field="*{confirmPassword}"/>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)